src/Persistence/MassTransit.S3.Storage/MassTransit.S3.Storage.csproj

```diff
@@ -0,0 +1,29 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="../../../signing.props" />
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;netstandard2.1;net6.0</TargetFrameworks>
```

Comment:
AWS doesn't have a netstandard2.1 package, so no need to build it. Match the targets for AmazonSQS.

Reply:
Removed